qa/run-tests.xml

```diff
@@ -73,19 +73,6 @@
   <property name="testoutput.zip.includes" value="testoutput/**,TEST*"/>
   <property name="profile" value=""/>
 
-  <condition property="modular.jdk.args" value="--add-modules=java.corba --upgrade-module-path ../ArjunaJTS/jtax/target/upgrademodulepath --add-exports java.corba/com.sun.corba.se.spi.orb=ALL-UNNAMED --add-exports java.corba/com.sun.corba.se.spi.ior=ALL-UNNAMED --add-exports java.corba/com.sun.corba.se.impl.ior=ALL-UNNAMED --add-opens=java.base/java.io=java.corba">
```

Comment:
The QA tests fail because `modular.jdk.args` is not set and ${modular.jdk.args} gets passed as the last element of the java command line so java wants it to be a classname to run. It should work if you just set `modular.jdk.args=` (ie the empty string) or maybe just get rid of `modular.jdk.args` completely - it get's added in run-tests.xml as an additional argument:

```<sysproperty key="additional.elements" value="${modular.jdk.args} @{additional.elements} ${objectStoreElements}"/>
```

Reply:
Thanks @mmusgrov I will put the PR on hold so it can be reviewed when it is passing

Reply:
ah, I reviewed it too early, sorry about that

Reply:
BTW, the QA_JTS worked - thanks for the help